docs/ExposePortsToInternet.md

```diff
@@ -6,41 +6,53 @@
 > You are viewing the calico-containers documentation for release **release**.
 <!--- end of master only -->
 
-# Expose Container Port to Host-Interface / Internet
+# Expose Container Port to Host Interface / Internet
 
 In the [Calico without Docker networking tutorial](calico-with-docker/without-docker-networking/README.md) 
 or the [Calico as a Docker network plugin tutorial](calico-with-docker/docker-network-plugin/README.md)
 we created containers and assigned endpoints (a container interface) to them. This is used for Container-
 To-Container communication.
 
-The example below shows how to expose a port of a container to the Host-Interface so this container is 
+The example below shows how to expose a port of a container to the host interface so this container is 
 reachable from outside / the internet.
 
 ## Why isn't the `-p` flag on `docker run` working as expected?
-If you connect containers to the docker0 interface, then calico would not be able to enforce security rules
-between workloads; all containers on the bridge would be able to communicate with each other.
+If you connect containers to the `docker0` bridge interface, Calico would not 
+be able to enforce security rules between workloads; all containers on the
+bridge would be able to communicate with one other.
 
-## How can i make a container reachable from the Internet (Host-Interface)?
-Let's say you want to expose port 80 of your container IP 192.168.0.1 to be reachable from the internet. 
-You can do that these two steps:
+When using Docker networking with Docker default IPAM, it is possible to call
```

Comment:
I think the ordering of this is wrong.

Basically if you use Docker IPAM then the Calico libnetwork plugin connects the container to the Docker bridge.  This allows the -p option to work (since that is handled in the bridge), but at the expense of full Calico policy isolation between containers connected to the same bridge.